examples/ecs-ec2/variables.tf

```diff
@@ -24,8 +24,8 @@ variable "coralogix_region" {
   description = "The region of the Coralogix endpoint domain: [Europe, Europe2, India, Singapore, US, US2, Custom]. If \"Custom\" then __custom_domain__ parameter must be specified."
   type        = string
   validation {
-    condition     = can(regex("^(europe|europe2|india|singapore|us|us2|custom)$", lower(var.coralogix_region)))
-    error_message = "Must be one of [Europe, Europe2, India, Singapore, US, US2, Custom]"
+    condition     = contains(["Europe", "Europe2", "India", "Singapore", "US", "US2", "Custom"], var.coralogix_region)
```

Comment:
Lets change it to  EU1, EU2, AP1, AP2, US1, US2, Custom and record in the Changelog that this is a breaking change

Reply:
Let me think about having coexistence... I think we should avoid breaking changes as a cultural habit. We do not know what is the blast radius, and do not want to be blamed for any problem no matter how small if probability of negative customer impact is non-zero. IMO.

Reply:
we can add support for certain point for both and than remove the old
the reason we need the names as i wrote them since this is how it present in our docs